stl/inc/format

```diff
@@ -2493,15 +2493,14 @@ _NODISCARD _OutputIt _Fmt_write(
         _THROW(format_error("const void* cannot be localized"));
     }
 
-    // Compute the bit width of the pointer (i.e. how many bits it takes to be represented).
-    // Add 3 to the bit width so we always round up on the division.
-    // Divide that by the amount of bits a hexit represents (log2(16) = log2(2^4) = 4).
-    // Add 2 for the 0x prefix.
-    auto _Width = 2 + static_cast<int>(_STD bit_width(reinterpret_cast<uintptr_t>(_Value)) + 3) / 4;
-
-    // Since the bit width of 0 is 0, special case it instead of complicating the math even more.
-    if (_Value == nullptr) {
-        _Width = 3;
+    // Since the bit width of 0 is 0x0, special-case it instead of complicating the math even more.

```

Comment:
What's the point of the 0x prefix? `std::bit_width<T>(T x)` returns T, so 0 is just as accurate as 0x0, and using 0x0 risks confusing it with the fact we're calculating the length of the string with `"0x"` prefixed.

Reply:
No idea actually. But my point stands: we don't need to call bit_width in that case